values_community_edition.yaml

```diff
@@ -37,6 +38,7 @@ gateway:
   service:
     type: LoadBalancer
     port: 443
+    controlApiPort: 9696
```

Comment:
This should be under a different service to avoid exposing the control api.

Reply:
separated it out (into the values.yaml) for its own gateway.api.service configuration (disabled by default)